client/my-sites/checkout/checkout-thank-you/domains/thank-you-content/domain-registration.tsx

```diff
@@ -24,33 +25,52 @@ const DomainRegistrationThankYouProps = ( {
 			selectedSiteSlug,
 		},
 		'REGISTRATION',
-		false
+		true
 	);
 
 	const returnProps: DomainThankYouProps = {
+		thankYouNotice: {
+			noticeTitle: translate(
+				'During setup your domain may be unreliable during the first 30 minutes.'
```

Comment:
This wording (especially the double use of "during") sounds a little awkward to me.

"It may take about 30 minutes for all of your new domain's settings to start working properly."

I would love to hear other ideas too.

Reply:
I've updated the copy with the one suggested by @hambai.